cpp/unreal_plugins/WheeledVehicle/Source/WheeledVehicle/VehiclePawn.h

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#pragma once
+
+#include <Eigen/Dense>
+
+#include <CoreMinimal.h>
+#include <GameFramework/Pawn.h>
+
+#include "VehiclePawn.generated.h"
+
+class UBoxComponent;
+class UCameraComponent;
+class UVehicleMovementComponent;
+class USkeletalMeshComponent;
+
+
+UCLASS()
+class WHEELEDVEHICLE_API AVehiclePawn : public APawn
```

Comment:
`AWheeledVehiclePawn`

Reply:
This was my first choice as well, but UE already has a class named `AWheeledVehiclePawn` (https://github.com/EpicGames/UnrealEngine/blob/5.2.0-release/Engine/Plugins/Experimental/ChaosVehiclesPlugin/Source/ChaosVehicles/Public/WheeledVehiclePawn.h).

Reply:
Makes sense. In that case, let's call the plugin `Vehicle` rather than `WheeledVehicle` and just use `Vehicle` as the prefix for everything for consistency. I think you're mostly doing this already except for the name of the plugin.

Reply:
Yeah, all classes in this plugin already begin with `Vehicle` prefix. Updated plugin name to `Vehicle`.